src/main/java/pl/bajerska/befindyourmeal/recipe/RecipeCriteria.java

```diff
@@ -0,0 +1,51 @@
+package pl.bajerska.befindyourmeal.recipe;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+
+
+public class RecipeCriteria {
+
+    ArrayList<String> ingredients = new ArrayList<String>();
+
+    @Enumerated(EnumType.STRING)
+    ArrayList<DietLabels> diet = new ArrayList<DietLabels>();
+
+    @Enumerated(EnumType.STRING)
+    ArrayList<HealthLabels> health = new ArrayList<HealthLabels>();
+
+    ArrayList<String> excluded = new ArrayList<String>();
```

Comment:
These should be definitely private fields. And, taking the first list as an example, You don't need to provide "default" setter setIngredients(ArrayList<String> ingredients), you can construct it in the constructor and offer addIngedient(String ingredient) method. :)